examples/life-expectancy/life-expectancy/requirements.txt

```diff
@@ -0,0 +1,4 @@
+inspect
```

Comment:
I think you are missing openpyxl (for the excel files). I had to install it locally to get the job running.

Reply:
Just updated the requirements.txt file with openpyxl